### ISSUE TITLE
A11y: calcite-modal

### ISSUE THREAD

**Issue by rslibed** [Esri/calcite-components#1756]
### Summary
Repoerted issues regarding `calcite-modal`.

### Issues
 - When a page first opens and a modal is opened on load, a screen reader should announce that there is a window open on the page

- The modal title does not get read aloud for screen readers

- The content in the modal does not get read by a screen reader so a non sighted user will not be able to read the content in the panels

**Comment by jcfranco** [comment]
Installed.

**Comment by jcfranco** [comment]
@rslibed Can you help verify this one on `@next` when it updates?

**Comment by rslibed** [comment]
Verified on v1.0.0-next.215. Title of modal is being read when modal is opened. Closing issue.

**Comment by jcfranco** [comment]
Thanks, @rslibed!
